src/index.ts

```diff
@@ -6,12 +6,13 @@ import BN from 'bn.js'
 
 interface ShiftPair {
   shift: number
+  extraShift: number
   x: string
   y: string
 }
 
 const SCI_NOTATION_REGEX = /^(-?\d*\.?\d*)e((?:\+|-)?\d+)$/
-
+const RECOMMENDED_DIVIDE_DIGITS = 77
```

Comment:
Add a comment giving more semantics to this constant.

Reply:
done